docs/concepts/explore-available-balancer-pools/stable-pool/stable-surge-pool.md

```diff
@@ -0,0 +1,60 @@
+---
+order: 2
+title: StableSurge Pool
+references:
+  - details: Stable Surge Hook
+    link: /reference/math/stablesurge-math
+  - details: Boosted Pools
+    link: /reference/math/stablesurge-math
+---
+
+# StableSurge Pools
+
+## Overview
+
+StableSurge Pools are a type of Stable Pool designed for assets that usually trade at nearly the same value or have a predictable exchange rate. What makes StableSurge Pools different is that they use a special feature—called a hook—to automatically adjust the swap fee based on how balanced the pool is during a trade.
+
+Here’s how it works:
+
+Normally, swaps pay a base fee (like 0.05%).
+
+But if the pool becomes unbalanced—meaning one asset makes up more than a set percentage—the fee starts increasing.
+
+This increase happens gradually (linearly) up to a maximum fee (like 0.50%) as the imbalance grows.
```

Comment:
```suggestion
This increase happens gradually, starting at the threshold level, and increasing linearly up to a maximum fee (like 50%) as the imbalance grows.
```
0.5% as a maximum seems really low. In tests and the code examples, we use close to 100%. I guess I'm not sure what they actually use; are those low numbers more realistic?